src/core/Akka/Actor/PipeToSupport.cs

```diff
@@ -27,28 +27,24 @@ public static class PipeToSupport
         /// <param name="sender">TBD</param>
         /// <param name="success">TBD</param>
         /// <param name="failure">TBD</param>
-        /// <returns>TBD</returns>
-        public static Task PipeTo<T>(this Task<T> taskToPipe, ICanTell recipient, IActorRef sender = null, Func<T, object> success = null, Func<Exception, object> failure = null)
+        /// <returns>A detached task</returns>
+        public static async Task PipeTo<T>(this Task<T> taskToPipe, ICanTell recipient, IActorRef sender = null, Func<T, object> success = null, Func<Exception, object> failure = null)
```

Comment:
So long as `PipeTo` is not itself `await`ed then the behavior will be identical to what it is in Akka.NET v1.4.27.

Going to let the unit tests demonstrate if I'm incorrect on this.

Reply:
so if we await a PipeTo, it will act as if we're doing an await to an Ask?

Reply:
So long as the `PipeTo` itself is not `await`ed, it will continue to run as a detached `Task` - `PipeTo` could have also been `await`-ed in current versions of Akka.NET too.

Reply:
But yes, if you `await` the `PipeTo` the `PipeTo` will also away on the antecedent `Task`, which is structurally identical to what it does now from a flow control perspective but it's semantically different because now `AsyncLocal` context propagates with it using local methods and context. 

Basically this means we don't have to close over the `Sender` anymore outside the `PipeTo` call as that's now evaluated in real time inside the `PipeTo` method prior to its `await`.

Reply:
> Will the exception result be different? As I remember, await will automatically unroll the AggregatedException thrown, so this can be a breaking change

Yeah this is what we discussed with @ismaelhamed - it definitely eliminates the `AggregateException` but still produces the correct `Status.Failure` message with the underlying `Exception` tucked inside the `Status.Failure.Cause` field so I think that should be fine. That property was never hard-coded to be an `AggregateException` in the first place, but it almost always was due to upstream `Task` handling.

So if that's too big of a change to fit into Akka.NET v1.4.28, I'd be fine moving it to v1.5